server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/exchanges/bittrex/BittrexExchange.java

```diff
@@ -67,6 +67,7 @@ private synchronized Exchange getExchange() {
         cryptoCurrencies.add(CryptoCurrency.BCH.getCode());
         cryptoCurrencies.add(CryptoCurrency.ETH.getCode());
         cryptoCurrencies.add(CryptoCurrency.LTC.getCode());
+        cryptoCurrencies.add(CryptoCurrency.BAY.getCode());
```

Comment:
Mirror the change in the batm-extensions.xml and please test it before pushing. You have to change the rate source as well as exchange.

Reply:
@rpanak-generalbytes I was told this has gone though extensive testing and should be ready to go. Can you let us know if this can get merged to the main chain and hopefully added in the next update? We would really like to start using Bitbay on our ATMs. Thanks for letting us know if there is anything else needed so we can proceed. Best, Alec

Reply:
As I already mentioned, the extensions file is missing the BAY support declaration for Bittrex (exchange as well as rate source). 
Maybe the code works now (it probably does if you tested it in a real BATM), but it might stop if we do stricter checking in the future.

Reply:
Hi. The Bay support declaration for Bittrex has been submitted. I would appricate the merge so that we can do live testing with other BATM operators. Thank you very much in advance and for your past comments. Very helpful.

Reply:
> (exchange as well as rate source)

You did only rate source. Please fix the exchange as well.

Reply:
Hi, I pushed the latest changes. Please merge

Reply:
Comment was just default one